src/js_ast.zig

```diff
@@ -7371,9 +7371,7 @@ pub const Macro = struct {
                 }
             };
 
-            // TODO: can change back to `return CallData.callWrapper(.{`
-            // when https://github.com/ziglang/zig/issues/16242 is fixed
-            return CallData.callWrapper(CallArgs{
+            return CallData.callWrapper(.{
```

Comment:
Please manually verify this works as expected. @dylan-conway probably remembers this one well

Reply:
To verify this one, you will need to verify the args in `runAsync` are the same as the input passed through `callWrapper`. If I remember correctly, some of the fields in the `CallArgs` struct would become garbage. Probably best way to test is run macros with 0-n args

Reply:
I should've added a test with this commit